From 77e895dcfadb156c9f378d26c9d0497ce2baf13f Mon Sep 17 00:00:00 2001
From: Rich Felker <dalias@aerifal.cx>
Date: Sat, 22 Apr 2017 20:40:09 -0400
Subject: [PATCH] add no-op POSIX_SPAWN_USEVFORK to spawn.h

the bit is reserved anyway for ABI-compat reasons; this documents it
and makes it so we can have posix_spawnattr_setflags check for flag
validity without hard-coding an anonymous bit value.
---
 include/spawn.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/spawn.h b/include/spawn.h
index f3e9e23c..bba57ce4 100644
--- a/include/spawn.h
+++ b/include/spawn.h
@@ -21,6 +21,7 @@ struct sched_param;
 #define POSIX_SPAWN_SETSIGMASK 8
 #define POSIX_SPAWN_SETSCHEDPARAM 16
 #define POSIX_SPAWN_SETSCHEDULER 32
+#define POSIX_SPAWN_USEVFORK 64
 #define POSIX_SPAWN_SETSID 128
 
 typedef struct {
-- 
2.13.0

